softdevice_controller/include/sdc.h

```diff
@@ -643,6 +643,17 @@ int32_t sdc_support_le_periodic_adv(void);
  */
 int32_t sdc_support_le_periodic_sync(void);
 
+/** @brief Support LE Power Control
+ *
+ * After this API is called, the controller will support the HCI commands
+ * related to the LE Power Control.
+ *
+ * @retval 0                Success
+ * @retval -NRF_EPERM       This API must be called before @ref sdc_cfg_set() or @ref sdc_enable().
+ * @retval -NRF_EOPNOTSUPP  LE Power Control is not supported.
+ */
+int32_t sdc_support_le_power_control(void);
```

Comment:
Is this part of DRGN-15903?
Otherwise, I think there might be a changelog entry missing.

Reply:
sdc_support_le_power_control is not fully supported yet, so not published in CHANGELOG yet.